amethyst_rendy/Cargo.toml

```diff
@@ -44,6 +44,7 @@ winit = "0.19"
 approx = "0.3"
 
 [features]
+default = ["vulkan"]
```

Comment:
doesn't this require non-vulkan users to use default-features = false?

Reply:
oops, used that for my local testing